pyaedt/maxwell.py

```diff
@@ -1465,20 +1523,24 @@ def enable_harmonic_force(
         Parameters
         ----------
         objects : list
-            Object list to enable force computation.
+            List of object names for force calculations.
         force_type : int, optional
-            Force Type. `0` for Objects, `1` for Surface, `2` for volumetric.
+            Force Type. ``0`` for Objects, ``1`` for Surface, ``2`` for volumetric.
```

Comment:
```suggestion
            Force type. Options are:
            
            - ``0``: objects
            - ``1``: surface
            - ``2``: volumetric
```
Should objects have an "s". It seems like it shouldn't....